lib/phoenix/html/tag.ex

```diff
@@ -0,0 +1,97 @@
+defmodule Phoenix.HTML.Tag do
+  @moduledoc ~S"""
+  Helpers related to producing HTML tags within templates.
+  """
+
+  alias Phoenix.HTML.Safe
+
+  @data_attrs [:method, :remote, :confirm]
```

Comment:
In fact we chose to moving away from handling an special set of data attributes on Rails. We just tell people to use the data attribute directly as `tag(:a, "My link", { data: { confirm: 'Something } })` instead of `tag(:a, "My link", confirm: 'Something })`